staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -328,6 +333,47 @@ func (g *gcePersistentDiskCSITranslator) GetCSIPluginName() string {
 	return GCEPDDriverName
 }
 
+func (g *gcePersistentDiskCSITranslator) RepairVolumeHandle(volumeHandle, nodeID string) (string, error) {
+	var err error
+	tok := strings.Split(volumeHandle, "/")
+	if len(tok) != volIDTotalElements {
+		return "", fmt.Errorf("volume handle has wrong number of elements; got %v, wanted %v", len(tok), volIDTotalElements)
```

Comment:
How brittle is this if we add more elements in the future?

Reply:
that would be very brittle. Also a breaking API change... it would be a pretty major change, but not only for this library..

Reply:
Could the library be better forward compatible by checking `>=`

Reply:
ok, made that change